src/js/model/Lotto.js

```diff
@@ -0,0 +1,21 @@
+import { LOTTO } from '../constants/constants';
+import { generateRandomNumber, shuffleNumber } from '../utils/number';
+
+export default class Lotto {
+  constructor() {
+    this.lottoNumbers = [];
+    this.generateLottoNumbers();
+  }
+
+  generateLottoNumbers() {
+    const candidate = Array(45)
+      .fill()
+      .map((element, index) => index + 1);
+
+    while (this.lottoNumbers.length < 6) {
+      shuffleNumber(candidate);
+      const pickedNumber = candidate.splice(0, 1)[0];
+      this.lottoNumbers.push(pickedNumber);
+    }
+  }
```

Comment:
```suggestion
    const candidate = Array(45)
      .fill()
      .map((element, index) => index + 1);
    shuffleNumber(candidate);
    ~~~~~~~~~~~
  }
```
한번만 섞고 잘라서 넣어줘도 충분하지 않을까 생각됩니다~!

Reply:
실제 로또 추첨기에서 `공이 계속 섞임 -> 하나씩 추출` 하는 걸 생각해서 구현했는데 프로그램으로 구현했을땐 불필요한 중복인 것 같기도 하네요! 섞고 앞에서 6개 잘라서 오는 방식으로 고쳐보겠습니다.